falabracman-info.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index 57a2158..960a710 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -2,9 +2,18 @@
+ name = falabracman
+ activity_version = 2
+ host_version = 1
+-bundle_id = org.laptop.community.falabracman
++bundle_id = org.sugarlabs.falabracmanActivity
```

Comment:
I recommend to use `org.sugarlabs.Falabracman`

Reply:
resolved